lib/block/operation.go

```diff
@@ -138,6 +138,10 @@ func (bo BlockOperation) NewBlockOperationSourceKey() string {
 	)
 }
 
+func (bo BlockOperation) LoadBody() (body transaction.OperationBody, err error) {
```

Comment:
@kfangw Please review this code. With using `BlockOperation.Body`, I could not find the way to access to original `OperationBody`, so I wrote new function, `transaction.UnmarshalOperationBodyJSON`(below), which is load `OperationBody` from `[]byte`.

Reply:
@spikeekips Looks good.
But please mind #456 in advance.
After boscon, let's do refactoring like that

Reply:
@kfangw Oops, I missed. I’ll ask your advice tomorrow.